README.md

```diff
@@ -139,3 +140,6 @@ Periodically, the group may have in-person meetings that coincide with a major
 conference. Please see the
 [meeting minutes doc](https://docs.google.com/document/d/1OVF68rpuPK5shIHILK9JOqlZBbfe91RNzQ7u_P7YCDE/edit#)
 for any future plans.
+
+[xml-format-working]: https://github.com/cloudevents/spec/blob/main/cloudevents/formats/xml-format.md
```

Comment:
I think this link needs to be updated.
Is the next link used?

Reply:
Correct